From f3763b75ddc8a6bf05b3091309c788a1481441bf Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 3 Mar 2015 15:38:54 +0100
Subject: [PATCH] ifcfg/write-ifcfg.sh: copy over all dhcp lease files

copy over the lease files, regardless of netroot succeeding with that
interface
---
 modules.d/45ifcfg/write-ifcfg.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/45ifcfg/write-ifcfg.sh b/modules.d/45ifcfg/write-ifcfg.sh
index e05d313..635a4a0 100644
--- a/modules.d/45ifcfg/write-ifcfg.sh
+++ b/modules.d/45ifcfg/write-ifcfg.sh
@@ -47,6 +47,8 @@ for netif in $IFACES ; do
         if [ -f /tmp/dhclient.$netif.lease ]; then
             strstr "$ip" '*:*:*' && echo "DHCPV6C=yes"
             echo "BOOTPROTO=dhcp"
+	    [ -s /tmp/dhclient.$iface.lease ] &&    cp /tmp/dhclient.$iface.lease    /tmp/net.$iface.lease
+	    [ -s /tmp/dhclient.$iface.dhcpopts ] && cp /tmp/dhclient.$iface.dhcpopts /tmp/net.$iface.dhcpopts
         else
             echo "BOOTPROTO=none"
         # If we've booted with static ip= lines, the override file is there
